### DISCUSSION TITLE
Is it ab bug?

### DISCUSSION THREAD

**Discussion by cracksilver** [automadcms/discussions#41] [General]
Hi there

If I try to link a picture with a short url like /categorie/product automad doesn't save it. If I use url like https://my.domain.com/categorie/product automad saves the url as it should. 

The same workflow with buttons work very well.

![grafik](https://github.com/user-attachments/assets/39fb0494-8eaf-4d4a-9a95-8986daa4e8a1)
![grafik](https://github.com/user-attachments/assets/76680e57-33a0-4357-b334-b3e086e233cd)

thx

**Comment by marcantondahmen** [comment]
Hi, yes that's a bug. Thanks for reporting. It will be fixed with the next release. 

Also please consider reporting bugs in the issue section in the future.

**Comment by cracksilver** [comment]
Thanks for confirming
